client/src/Routes.jsx

```diff
@@ -1,16 +1,49 @@
-import React, { Fragment, Suspense, lazy } from 'react';
+import React, { Suspense, lazy } from 'react';
 import { HashRouter, Route, Switch } from 'react-router-dom';
+// eslint-disable-next-line camelcase
+import jwt_decode from 'jwt-decode';
+import { Provider } from 'react-redux';
 
 import AppContainer from './components/AppContainer';
 import Loader from './components/Shared/Loader/Loader';
 import Home from './components/Home/Home';
+import setAuthToken from './utils/setAuthToken';
+
+import store from './store/store';
+import { logoutUser, setCurrentUser } from './actions/authActions';
+import { getHabits } from './actions/habitActions';
+
+// Check for auth token
+if (localStorage.jwtToken) {
```

Comment:
This logic, as well as Provider wrapper, usually live in the index.js file. It's not big deal, probably just me nitpicking 😄 	But this file should be reserved only for routing logic.

Reply:
The nitpicking is appreciated -- separation of concerns makes complete sense in this case 😄